keps/sig-instrumentation/383-new-event-api-ga-graduation/README.md

```diff
@@ -439,6 +449,201 @@ List Events from the NamespaceSystem with field selector `reportingController =
 
 List all Event with field selector `regarding.name = podName, regarding.namespace = podNamespace`, and `related.name = podName, related.namespace = podNamespace`. You need to join results outside of the kubernetes API.
 
+## Production Readiness Review Questionnaire
+
+### Feature enablement and rollback
+
+_This section must be completed when targeting alpha to a release._
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [ ] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name:
+    - Components depending on the feature gate:
+  - [x] Other
+    - Describe the mechanism:
+
+      (1) The API itself can be enabled / disabled at kube-apiserver level by using `--runtime-config` flag;
+      (2) For the use of API, we have a fallback mechanism instead of using a feature gate. That is, we simply fallback to the old Event libraries if the API is diabled.
+
+    - Will enabling / disabling the feature require downtime of the control
+      plane?
+
+      No.
+
+    - Will enabling / disabling the feature require downtime or reprovisioning
+      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
+
+      No.
+
+* **Does enabling the feature change any default behavior?**
+  Any change of default behavior may be surprising to users or break existing
+  automations, so be extremely careful here.
+
+  While the graduation of the API itself doesn't change default behavior, migration of individual components does, as the events will be reported differently.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we rollback
+  the enablement)?**
+  Also set `rollback-supported` to `true` or `false` in `kep.yaml`.
+  Describe the consequences on existing workloads (e.g. if this is runtime
+  feature, can it break the existing applications?).
+
+  Yes. If the new Event API is disabled, it will fallback to the original one.
```

Comment:
Please also add a sentences that
1. the new events are roundtrippable with the old corev1.Events. (for this rollback)
2. rollback of client-library use may not be possible if individual components won't implement it (i.e. they only fallback to the old API if the new API is disabled, if there is bug in the client-library, there is no way to fallback as of now)

Reply:
Done.